documentation/docs/steps/apiKeyValueMapUpload.md

```diff
@@ -0,0 +1,50 @@
+# ${docGenStepName}
+
+## ${docGenDescription}
+
+* This step ApiKeyValueMapUpload helps you to store one or more key value pairs of data in a grouping called a map, or key value map.
+
+* A typical flow would look something like
+
+You have to store API management service key in jenkins server as secret text. this is obtained from the SAP BTP sub account cockpit, under instance and subscriptions for API management under API plan. then you create new jenkins pipeline, which consumes the ApiKeyValueMapUpload piper step. you can execute the pipeline and validate step exection results as explained in the blog [Integration Suite Piper commands](https://blogs.sap.com/2022/01/05/working-with-integration-suite-piper-commands/)
```

Comment:
"Get an API management service key from ..."
"Store the key ...."
"Create a Jenkins pipeline..."
"Execute...

Capital letters at the beginning of a sentence!

Reply:
fixed